Sming/Core/Network/HttpServer.h

```diff
@@ -29,6 +29,7 @@ typedef struct {
 	int keepAliveSeconds = 0;	  ///< default seconds to keep the connection alive before closing it
 	int minHeapSize = -1;		   ///< min heap size that is required to accept connection, -1 means use server default
 	bool useDefaultBodyParsers = 1; ///< if the default body parsers,  as form-url-encoded, should be used
+	bool closeOnBadBodyContent = false;
```

Comment:
Please, make the default value to be `true`. On embedded device the default behaviour should be a bit more strict and trying to preserve resources. Further in the samples the users can adjust this so that for example Firefox does not show an error message. And in the sample(s) the contributor(s)  can and should add additional comments describing the advantages and disadvantages of having this set to `true`.

Reply:
Done, but I have some mixed feelings about this. On the one hand, I understand your reasoning, especially for M2M applications. However, for HMI-focused applications, this default may lead to some unexpected and hard to track down behaviour...

Anyway, either default is fine for me, as long as it allows to resolve the open issue in #1817.